cloud/pkg/cloudstream/streamserver.go

```diff
@@ -140,7 +139,7 @@ func (s *StreamServer) getContainerLogs(r *restful.Request, w *restful.Response)
 		klog.Infof("Delete %s from %s", logConnection.String(), session.String())
 	}()
 
-	if err := logConnection.Serve(); err != nil {
+	if err = logConnection.Serve(); err != nil {
 		err = fmt.Errorf("apiconnection Serve %s in %s error %v",
```

Comment:
If we don't remove `:` above, the err returned maybe nil, right?

Reply:
yes

Reply:
But the err here has any usage?

Reply:
`WriteHeader` in the defer func
https://github.com/kubeedge/kubeedge/blob/bf51f6d2db9d83559b3f1da79e5f724492ddc433/cloud/pkg/cloudstream/streamserver.go#L100-L109

Reply:
Oh, lgtm